docs/manifest.md

```diff
@@ -1,21 +1,25 @@
-# Manifest definition file
+# Manifest store file
 
-The manifest definition file is a JSON formatted file with a `.json` extension.
-Relative file paths are interpreted as relative to the location of the definition file unless you specify a `base_path` field.
+The manifest store file is a JSON file with `.json` extension. Relative file paths are interpreted as relative to the location of the definition file unless you specify a `base_path` field.
 
-## Example manifest definition file
+## Example
 
-Here's an example of a manifest definition that inserts a CreativeWork author assertion. Copy this JSON int a file to use as a test manifest.
+The example below is a snippet of a manifest definition that inserts a CreativeWork author assertion. This example uses the default testing certificates in the [sample folder](https://github.com/contentauth/c2patool/tree/main/sample) that are also built into the c2patool binary.   Copy this JSON into a file to use as a test manifest. 
 
-It is important to provide a value for the Time Authority URL (the `ta_url` property) to have a valid timestamp on the claim.
+**NOTE**:  When you don't specify a key or certificate in the manifest `private_key` and `sign_cert` fields, the tool will use the built-in key and cert. You'll see a warning message, since they are meant for development purposes only. For actual use, provide a permanent key and certificate in the manifest definition or environment variables; see [Creating and using an X.509 certificate](x_509.md). 
 
-The default certificates in the [sample folder](https://github.com/contentauth/c2patool/tree/main/sample) are built into the c2patool binary. This example uses the default testing certs. You will see a warning message when using them, since they are meant for development purposes only.
+The following manifest properties are specific to c2patool and used for signing manifests:
 
-**NOTE**: Use the default private key and signing certificate only for development.
-For actual use, provide a permanent key and cert in the manifest definition or environment variables; see [Creating and using an X.509 certificate](x_509.md).
+- `alg`: Signing algorithm to use. See [Creating and using an X.509 certificate](x_509.md) for possible values. Default: `es256`.
+- `private_key`: Private key to use. Default: `es256_private.key`
+- `sign_cert`: Signing certificate to use. Default: `es256_certs.pem`
+- `ta_url`:  Time Authority URL. Required for a valid time-stamp. For example, `http://timestamp.digicert.com`.
```

Comment:
The ta_url is not required for a valid time stamp, but if you don't have it, and the cert expires, there's no way to know if the manifest was signed while the cert was valid. So unless your cert never expires and is never revoked, it is a really good idea to have a timestamp. However, the ta_url requires a live online connection and that may not always be possible.